src/ReactDataGrid.js

```diff
@@ -322,15 +317,17 @@ const ReactDataGrid = React.createClass({
 
   onDragStart(e: SyntheticEvent) {
     let idx = this.state.selected.idx;
-    if (idx > -1) {
+    // To prevent dragging down/up when reordering rows.
+    const isDragging = e && e.target && e.target.className;
```

Comment:
when we are dragging rows using react-dnd we are also dragging, can this change to for example `isViewportDrag`